runtime.sh

```diff
@@ -6,18 +6,18 @@
 # Initialise CMD as empty
 CMD=""
 
-# If an arg is defined, start contaier with bash
+# If an arg is defined, start container with bash
 if [ -n "$1" ]; then
     CMD="bash"
 fi
 
 # Build docker image only up to base stage
 DOCKER_BUILDKIT=1 docker build \
--t imu_humble \
--f Dockerfile --target runtime .
+    -t imu_humble \
```

Comment:
Please change this for `av_imu_microstrain_launch_humble`

Reply:
Done